app/views/items/_item.html.erb

```diff
@@ -1,3 +1,4 @@
 <div class="col-md-6">
     <%= item.name %>
+    <%= link_to "", item, method: :delete, class: 'glyphicon glyphicon-ok' %>
```

Comment:
do a rake routes but i think you want something like: 

link_to "", [current_user, item], method: :delete, class:  'glyphicon glyphicon-ok'

so your items are nested under your user - which is why simply putting item as the path doesn't work